include/ui/input-text-model.hpp

```diff
@@ -42,10 +61,16 @@ class InputTextModel : QObject {
   Q_INVOKABLE QString getStart();
 
  private:
-  QQmlContext* context;
-  int start;
-  unsigned int maximumLength;
-  MemoryAccess memoryAccess;
+  QQmlContext* _context;
+  int _start;
+  unsigned int _maximumLength;
```

Comment:
Here, for example, I would use `using length_t = unsigned int` and then only refer to that type instead of `unsigned int`